### PR TITLE
Personal portfolio for mobile version

### PR DESCRIPTION
**Pull request by kemigabocatherine**
This pull request contains a number of files;
    a .html file with the structure of my personal portfolio
    a .css file with styling to improve the appearance of my personal portfolio
    a README.md file describing the basics of the project
    a .gitignore file containing files that should be ignored by git
    linters to test for any errors.

### CODE REVIEW

File: portfolio.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            <p>LET’S CONNECT</p>
+        </div>
+        <br>
+        <div class="contact">
+            <a ><img class="twitter" src="images/Twitter@2x.png" alt="Twitter"></a>
+            <a ><img class="linkedin" src="images/Linkedin icon@2x.png" alt="Linkedin"></a>
+            <a ><img class="email" src="images/Mail@2x.png" alt="email"></a>
+            <a ><img class="github" src="images/Github@2x.png" alt="Github"></a>
+            <a ><img class="vector" src="images/Vector@2x.png" alt="Vector"></a> 
+        </div>
```
[the remark this answers is not in the document]
Comment (kemigabocatherine) [reply portfolio.html:39]: When I do this, are there any specific changes I should expect to occur? 
OR Is it for improving the readability of the code?

File: portfolio.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        <div class="connect">
+            <p>LET’S CONNECT</p>
+        </div>
+        <br>
+        <div class="contact">
+            <a ><img class="twitter" src="images/Twitter@2x.png" alt="Twitter"></a>
+            <a ><img class="linkedin" src="images/Linkedin icon@2x.png" alt="Linkedin"></a>
+            <a ><img class="email" src="images/Mail@2x.png" alt="email"></a>
+            <a ><img class="github" src="images/Github@2x.png" alt="Github"></a>
+            <a ><img class="vector" src="images/Vector@2x.png" alt="Vector"></a>
```
[the remark this answers is not in the document]
Comment (kemigabocatherine) [reply portfolio.html:38]: Thank you so much for the resource above

File: portfolio.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    text/css"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+
+</head>
+<body>
+    <header class="toolbar">
+        <nav>
+            <img src="images/Logo Placeholder.png" alt="My Logo"/>
```
[the remark this answers is not in the document]
Comment (mikethreels) [reply portfolio.html:18]: Hi Cathy, 
Only the logo will be enough 😄

**Comment by mimipeshy** [comment]
## Hello @kemigabocatherine  👋,

_This is an additional review as a result of the second opinion request._

Good job so far for the good work you have been doing. As the previous reviewer mentioned, it's better to wrap the list in `ul` rather than a `div`. This is due to semantic and readability as you have said. This will help screen readers interpret what is there. You might need to style it more according to Figma design but its readability will be greatly improved.

Kindly submit another review in your dashboard when done!

Keep rocking 🚀!

**Review by mikethreels — APPROVED** [review]
Hi Cathy, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 :
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**